packages/terra-framework-docs/src/terra-dev-site/doc/table/About.1.doc.mdx

```diff
@@ -14,13 +14,48 @@ Terra Table is a structural component used to display tabular data. Table provid
 
 Tables are often used to present a combination of information and interactive widgets. Since a table is not a widget, each widget contained in a table is a separate stop in the page tab sequence. 
 If the number of widgets is large, replacing the table with a grid can dramatically reduce the length of the page tab sequence because a grid is a composite widget that can contain other widgets.
 
+<DefaultTable />
+
+  ### Features
+
+  * [Zebra Striping](/components/cerner-terra-framework-docs/table/examples/zebra-striped-table)
+  * [Pinned Columns](/components/cerner-terra-framework-docs/table/examples/zebra-striped-table)
+  * [Sections](/components/cerner-terra-framework-docs/table/examples/table-with-sections)
+  * [Single Row Selection](/components/cerner-terra-framework-docs/table/examples/table-single-row-selection)
+  * [Multiple Row Selection](/components/cerner-terra-framework-docs/table/examples/table-multiple-row-selection)
+  * [Headerless Table](/components/cerner-terra-framework-docs/table/examples/table-without-headers)
+  * [Column Sorting](/components/cerner-terra-framework-docs/table/examples/sortable-table)
+  * [Column Error Indicator](/components/cerner-terra-framework-docs/table/examples/table-column-states)
+
+## Examples
+|Link to Example| Description|
+|-|-|
+|[Default Table](/components/cerner-terra-framework-docs/table/examples/default-table)|An example demonstrating a basic table.|
+|[Zebra Striped Table](/components/cerner-terra-framework-docs/table/examples/zebra-striped-table)|An example demonstrating the ability enable zebra striping.|
+|[Pinned Columns](/components/cerner-terra-framework-docs/table/examples/pinned-columns)|An example demonstrating _pinned_ columns (frozen leftmost columns) and _overflow_ columns.|
+|[Sortable Table](/components/cerner-terra-framework-docs/table/examples/sortable-table)|An example demonstrating how to enable sorting within a table.|
+|[Table Column States](/components/cerner-terra-framework-docs/table/examples/table-column-states)|An example demonstrating column states and interactions.|
+|[Table with Focusable Cells](/components/cerner-terra-framework-docs/table/examples/table-focusable-cell)|An example demonstrating tab stop behavior in a table with interactable elements.|
+|[Single Row Selection](/components/cerner-terra-framework-docs/table/examples/table-single-row-selection)|An example demonstrating a table with single row selection mode.|
+|[Multiple Row Selection](/components/cerner-terra-framework-docs/table/examples/table-multiple-row-selection)|An example demonstrating a table with multiple row selection mode.|
+|[Table with Sections](/components/cerner-terra-framework-docs/table/examples/table-with-sections)|An example demonstrating a table with sections.|
+|[Table Without Headers](/components/cerner-terra-framework-docs/table/examples/table-without-headers)|An example demonstrating a table without headers.|
+
```

Comment:
As mentioned previously, we should bubble up the prop names used for the examples into this Example table.  You did a good job in the individual examples explaining them in my detail.